### PR TITLE
add only url() part of background, with tests and image-set support

### PR DESCRIPTION
**Pull request by Razunter**

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -19,6 +19,20 @@ module.exports = (opts = {}) => {
...
+   * Process background value to remove unnecessary parts
+   * @param {string} value - background value
+   */
+  function processBackgroundValue(value) {
+    // check for image-set
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/image/image-set
+    if (value.includes('image-set')) {
+      return /(image-set.*\))/gm.exec(value)[0] ?? value
+    } else {
+      return /(url.*\))/gm.exec(value)[0] ?? value
```
Comment (ai) [inline plugin.js:32]: Should we use proper parser here for the case of having `)` in file name? https://github.com/TrySound/postcss-value-parser

Comment (Razunter) [reply plugin.js:32]: Probably would be better, but it passed the test with `url(\'./image().png\') 50% 50%`

Comment (ai) [reply plugin.js:32]: Nice

**Comment by ai** [comment]
Thanks. Released in 0.8.

**Pull request merged**